modules/integration/ddb-to-opensearch/stack.py

```diff
@@ -52,6 +53,8 @@ def __init__(
         os_security_group = ec2.SecurityGroup.from_security_group_id(
             self, f"{dep_mod}-os-sg", opensearch_sg_id, allow_all_outbound=True
         )
+        # Allow ingress
+        os_security_group.add_ingress_rule(peer=os_security_group, connection=ec2.Port.all_traffic())
```

Comment:
I think I'd rather fix this in the upstream opensearch IDF module but let me know.